packages/api/src/platforms/vtex/resolvers/collection.ts

```diff
@@ -11,9 +11,16 @@ const isBrand = (x: any): x is Brand => x.type === 'brand'
 const isPortalPageType = (x: any): x is PortalPagetype =>
   typeof x.pageType === 'string'
 
+const slugify = (root: Root) =>
+  isBrand(root)
+    ? baseSlugify(root.name)
+    : isPortalPageType(root)
+    ? new URL(`https://${root.url}`).pathname.slice(1)
+    : new URL(root.url).pathname.slice(1)
+
```

Comment:
What do you think about readability of this code? Remember that those conditions can increase and this ternary will be hard to read.
```suggestion
const slugify = (root: Root) => {
  if (isBrand(root)) {
    return baseSlugify(root.name)
  }
  
  if (isPortalPageType(root)) {
    return new URL(`https://${root.url}`).pathname.slice(1)
  }
  
  return new URL(root.url).pathname.slice(1)
}

```

Reply:
nice!